examples/nlohmann-json/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+cmake_minimum_required (VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(dowload-nlohmann-json)
+
+hunter_add_package(nlohmann-json)
+find_package(nlohmann-json)
+
+add_executable(main main.cpp)
+
+target_link_libraries(main
+  nlohmann-json::nlohmann-json)
+set_target_properties(main PROPERTIES
+  COMPILE_FLAGS ${NLOHMANN_JSON_DEFINITIONS})
```

Comment:
if compile flags are usage requirement we can add them in `*Config.cmake` by [INTERFACE_COMPILE_OPTIONS](http://www.cmake.org/cmake/help/v3.0/prop_tgt/INTERFACE_COMPILE_OPTIONS.html) property. Or better by [target_compile_options](http://www.cmake.org/cmake/help/v3.0/command/target_compile_options.html).